docs/contributing/checklists.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: console
+
+.. |X| raw:: html
+
+    <input type="checkbox">
+
+Pull Request Checklists
+-----------------------
+
+This document contains checklists for dealing with PRs. More general PR information is available at :ref:`PRs`.
+
+.. _reviewingPRs:
+
+Reviewing Pull Requests
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When reviewing a PR, do the following:
+
+* |X| Make sure it is coming from ``issues/XXXX-fix-the-thing`` in the Toil repo, or from an external repo.
+    * |X| If it is coming from an external repo, make sure to pull it in for CI with::
+        
+        contrib/admin/test-pr otheruser theirbranchname issues/XXXX-fix-the-thing
+    * |X| If there is no associated issue, `create one <https://github.com/DataBiosphere/toil/issues/new>`_.
+* |X| Read through the code changes. Make sure that it doesn't have:
+    * |X| Addition of trailing whitespace.
```

Comment:
This might be better as a `make` command and linting step, rather than a manual check.

We could probably just remove `W291` here: https://github.com/DataBiosphere/toil/blob/master/Makefile#L248 and add the linting line in `gitlab-ci.yml`.  I'll go ahead and make a PR.

Reply:
Is there a way to get flake8 to ignore preexisting problems and only fail when new problems are added by a commit?

Reply:
Yes, check out diff-cover https://diff-cover.readthedocs.io/en/latest/README.html

Reply:
I'll leave this in the checklist for now; setting up an automated version of this constraint is a new issue:  #3443